Khan/wonder-blocks
# WB-646: Add Live region to announce available options

## Summary:

Fixes an issue with iOS that was not announncing properly the total of options.

- Created new wrapper for announcing the options count when the dropdown menu
(`listbox`) is opened.
- Added unit tests.

NOTE: There's a known issue with the `SingleSelect` not being able to use the correct text when there's a single item in the list. This should be fixed after we allow passing translated strings to this component.


Issue: WB-646

## Test plan:

Using an iPad or iPhone:

1. Activate `Voice Over`.
2. Navigate to any of the dropdown stories.
3. Click on the dropdown opener.
4. Verify that VO announces the # of options (e.g. `9 items`).
5. Focus on the search field and filter some elements by typing in `o`.
6. Verify that VO also announces the filtered # of options (e.g. `4 items`).

https://user-images.githubusercontent.com/843075/168170382-300ea9d3-fda0-4c7d-807f-632385429590.mov
